uavcan/equipment/gnss/1063.Fix2.uavcan

```diff
@@ -0,0 +1,86 @@
+#
+# GNSS ECEF and LLA navigation solution with uncertainty.
+#
+
+uavcan.Timestamp timestamp         # Global network-synchronized time, if available, otherwise zero
+
+#
+# Time solution.
+# Time standard (GPS, UTC, TAI, etc) is defined in the field below.
+#
+uavcan.Timestamp gnss_timestamp    # GNSS timestamp, if available, otherwise zero
+
+#
+# Time standard used in the GNSS timestamp field.
+#
+uint3 GNSS_TIME_STANDARD_NONE = 0  # Time is unknown
+uint3 GNSS_TIME_STANDARD_TAI  = 1
+uint3 GNSS_TIME_STANDARD_UTC  = 2
+uint3 GNSS_TIME_STANDARD_GPS  = 3
+uint3 gnss_time_standard
+
+void13   # Reserved space
+
+#
+# If known, the number of leap seconds allows to perform conversions between some time standards.
+#
+uint8 NUM_LEAP_SECONDS_UNKNOWN = 0
+uint8 num_leap_seconds
+
+#
+# Position and velocity solution
+#
+int37 longitude_deg_1e8            # Longitude degrees multiplied by 1e8 (approx. 1 mm per LSB)
+int37 latitude_deg_1e8             # Latitude degrees multiplied by 1e8 (approx. 1 mm per LSB on equator)
+int27 height_ellipsoid_mm          # Height above ellipsoid in millimeters
+int27 height_msl_mm                # Height above mean sea level in millimeters
+
+float16[3] ned_velocity            # NED frame (north-east-down) in meters per second
+
+#
+# Fix status
+#
+uint6 sats_used
+
+uint2 STATUS_NO_FIX    = 0
+uint2 STATUS_TIME_ONLY = 1
+uint2 STATUS_2D_FIX    = 2
+uint2 STATUS_3D_FIX    = 3
+uint2 status
+
+#
+# GNSS Mode
+# 
+uint3 MODE_SINGLE      = 0
+uint3 MODE_DGPS        = 1
+uint3 MODE_RTK         = 2
+uint3 MODE_PPP         = 3
+uint3 mode
+
+#
+# GNSS Sub mode
+#
+
+uint5 SUB_MODE_DGPS_OTHER    = 0
+uint5 SUB_MODE_DGPS_SBAS     = 1
+
+uint5 SUB_MODE_RTK_FLOAT     = 0
+uint5 SUB_MODE_RTK_FIXED     = 1
+
+uint5 sub_mode
+
+#
+# Precision
+#
+float16 pdop
+
+float16[<=21] covariance    # Position and velocity covariance. The upper-right
+                            # triangle of a 6x6 symmetric matrix, represented in
+                            # row-major order. Units are m^2 for position, (m/s)^2
+                            # for velocity and m^2/s for position/velocity.
```

Comment:
I'm not completely sure of what you mean. Won't an array with 36 float elements use 36*2 bytes, and an array with 21 float elements use 21*2 bytes bandwidth? Are you suggesting the parser understands that it's a symmetric matrix and removes the 15 redundant elements? How does it then differ between a 21 element array, 6x6 full matrix, and a 6x6 upper triangular matrix?

There might be something I'm missing here.